sparkmagic/sparkmagic/kernels/kernelmagics.py

```diff
@@ -451,6 +453,10 @@ def get_session_settings(self, line, force):
     def _override_session_settings(settings):
         conf.override(conf.session_configs.__name__, settings)
 
+    @staticmethod
+    def _merge_required_session_settings():
+        conf.override_required()
```

Comment:
nit: should we standardize on `merge` or `override` for naming all of these functions? It seems more intuitive that `_merge_required_session_configs` called `conf.merge_required_session_configs`

Reply:
i like merge as well. to be honest, i have no idea why i went with override here